src/shared/utilities/vsCodeUtils.ts

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+export const localize = nls.loadMessageBundle()
+
+import { ErrorOrString, getLogger, Logger } from '../logger'
+
+// ------- Experimental combined output channel & traditional logger  --------------
+
+type LogLevel = 'debug' | 'info' | 'warn' | 'error'
```

Comment:
Should we use the same levels as `shared/logger`? That includes these four, plus `'verbose'`. Ideally there should be a single source of truth for this type.

Reply:
Oh ya thanks meant to import `shared/logger.LogLevel` but it didn't exist. Added `shared/logger.LogLevel` and used it.
✔️

Reply:
Added support for verbose.